.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,16 +27,12 @@ Steps to reproduce the behavior:
 A clear and concise description of what you expected to happen.
 
 ## Documentation
-If applicable, add screenshots or logfiles to help explain your problem.
+If applicable, add screenshots or logs to help explain your problem.
 
-Logfiles can be found in different places depending on platform:
+Logs can be found in different places depending on platform:
```

Comment:
Can maybe also mention that the log files can be opened with any standard text editor. Some windows users have not understood that files ending with ".log" is just a standard text file.